README.md

```diff
@@ -39,6 +39,15 @@ The alpha has support for the latest version of Node.js (v9) and npm (v5).
 npm install strapi@alpha -g
 ```
 
+##### Requirements
+```
+NodeJS >= 9
```

Comment:
`NodeJS >= 9` includes Node 10? 🤔

I know is silly, but this is read as if all the databases was required. Maybe something like is could be better:

```
+ NodeJS => 9.x
+ _One of this databases_:
  - MongoDB >= 10
  - ... etc.
```

🤷‍♂️

Reply:
Yeah I believe that should just state node 10 considering its going into LTS in October, and v9 was just a beta patch branch for v8

And yeah for the DB since the experience is more or less the same between databases it should go along with that